library/std/src/io/error.rs

```diff
@@ -103,6 +103,12 @@ pub enum ErrorKind {
     /// The connection was reset by the remote server.
     #[stable(feature = "rust1", since = "1.0.0")]
     ConnectionReset,
+    /// The remote host is not reachable.
+    #[stable(feature = "io_error_unix_more", since = "1.51.0")]
+    HostUnreachable,
```

Comment:
This looks like windows [`ERROR_HOST_UNREACHABLE`](https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--1000-1299-)

Reply:
Oh, for sockets there's also [`WSAEHOSTUNREACH`](https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--9000-11999-)